app/views/posts/show.html.erb

```diff
@@ -1 +1,13 @@
-<h2>This is the show page for posts</h2>
+<h1> show post for a given user</h1>
+<div>
+  <div>
+    <h3> <%=@post.title%> by <%= @user.name%> </h3>
+    <p> <%= @post.text%> </p>
+    <p> Comments: <%= @post.comments_counter %>, Likes: <%= @post.likes_counter %> </p>
+  </div>
+  <div>
+    <% @post.recent_five_comments.each do |comment| %>
+      <p> <%= comment.user.name%>: <%= comment.text%><p>
+    <% end %>
+  </div>
+</div>
```

Comment:
-[ ] Kindly style your post show page to match the wireframe: [https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=GHSAT0AAAAAABY4RVENF5O6IYX2BVYE7R7GY4J3RNQ](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=GHSAT0AAAAAABY4RVENF5O6IYX2BVYE7R7GY4J3RNQ)